_layouts/default.html

```diff
@@ -19,7 +19,7 @@
             <hr class="pldoc-divider sr-only">
 
             <div class="pldoc-wrapper-content-main">
-                <main class="pldoc-content pldoc-content-main" id="content-main">
+                <main class="pldoc-content pldoc-content-main" id="content-main" aria-label="Main">
```

Comment:
@cptvitamin, is this the standard convention? If so, I'm happy to follow it. Otherwise, I'd prefer that we rely on the semantic value of ``<main>`` by itself.